js/stores/appStore.js

```diff
@@ -328,6 +328,9 @@ function handleChangeSettingAction (settingKey, settingValue) {
 }
 
 const handleAppAction = (action) => {
+  const ledger = require('../../app/ledger')
+  ledger.boot()
```

Comment:
@diracdeltas I took it out and it seems to work fine but would love to hear @mrose17 weigh in!